packages/manager/modules/pci/src/projects/project/storages/databases/database.service.js

```diff
@@ -149,14 +149,20 @@ export default class DatabaseService {
     plan,
     version,
     flavor,
-    restApi = false,
   ) {
     return this.$http
       .put(`/cloud/project/${projectId}/database/${engine}/${databaseId}`, {
         description,
         plan,
         version,
         flavor,
+      })
+      .then(({ data }) => data);
+  }
+
+  putRestApi(projectId, engine, databaseId, restApi) {
+    return this.$http
+      .put(`/cloud/project/${projectId}/database/${engine}/${databaseId}`, {
         restApi,
       })
       .then(({ data }) => data);
```

Comment:
I think could be better like this :

```suggestion
  updateputDatabaseEngineProperties(projectId, engine, databaseId, properties) {
    return this.$http
      .put(`/cloud/project/${projectId}/database/${engine}/${databaseId}`, properties)
      .then(({ data }) => data);
```